### PR TITLE
Fixed broken unittest

### PR DESCRIPTION
**Pull request by ekoval**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: tests/partners/test_integration.py [hunk 1]
```diff
@@ -27,16 +27,12 @@ def setUp(self):
...
-        del self.result['partnerId']
-        del self.result['partnerToken']
-        del self.result['whiteLabelSettings']['urlLinks']
 
     def test_partner_is_returned(self):
         expected_result = dict(self.partner_data)
         expected_result['name'] = self.partner_name
-        expected_result['remoteId'] = 'remote_{}'.format(self.partner_name)
-        self.assertEqual(self.result, expected_result)
+        import pdb;pdb.set_trace()
```
Comment (snitch) [inline tests/partners/test_integration.py]: E231 missing whitespace after ';'
E702 multiple statements on one line (semicolon)

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: tests/base.py [hunk 2]
```diff
@@ -11,6 +11,18 @@
 from sitewit.services import SitewitService
 
 
+def dictcontainsdict(d1, d2):
```
Comment (RayeN) [inline tests/base.py]: What if we make this assertation? e.g. `assert_sub_dict`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # [Changelog](https://github.com/yola/sitewit/releases)
 
+## DEV
+
+* Fixed broken unittest
```
Comment (RayeN) [inline CHANGELOG.md]: Mention in changelog that you did tests more flexible?

**Comment by RayeN** [comment]
Can you please test dynamic partner creation during account creation as well? (since we're dealing with tests anyway)

File: tests/base.py (hunk 2 above)
Comment (ekoval) [inline tests/base.py]: I don't want to mix logic with asserts related to unittests.

File: tests/base.py [hunk 4]
```diff
@@ -11,6 +11,18 @@
...
+    for (key, value) in d2.items():
+        if key not in d1:
+            return False
+        if isinstance(value, dict):
+            return dictcontainsdict(d1[key], value)
+        if d1[key] != value:
+            return False
```
Comment (ekoval) [inline tests/base.py]: What value will we use instead of "None"?

File: CHANGELOG.md (hunk 3 above)
Comment (ekoval) [inline CHANGELOG.md]: Ok

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: sitewit/services.py [hunk 5]
```diff
@@ -437,21 +437,26 @@ def create_partner(self, name, address, settings, remote_id=None):
...
 
-    def get_partner(self, subpartner_id):
+    def get_partner(self, subpartner_id=None, remote_subpartner_id=None):
         """Get subpartner by subpartner id.
 
         Get subpartner by subpartner id.
 
         Args:
             subpartner_id (str): Subpartner ID.
+            remote_subpartner_id (str): Renote Subpartner ID.
```
Comment (RayeN) [inline sitewit/services.py]: typo, should be `Remote...`?

File: tests/base.py (hunk 2 above)
Comment (RayeN) [inline tests/base.py]: but that's a helper for unittests, nope?  Can we at least separate words in the function name with `_`

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: tests/base.py (hunk 4 above)
Comment (ekoval) [inline tests/base.py]: this will be smart, I'd prefer to keep it straightforward

File: tests/base.py (hunk 2 above)
Comment (ekoval) [inline tests/base.py]: This is something common, not directly related to unittests itself.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**